lxd/instance_post.go

```diff
@@ -288,7 +324,7 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 			return operations.OperationResponse(op)
 		}
 
-		if targetNode != "" {
+		if target != "" {
```

Comment:
I think this should be `targetMemberInfo != nil` so that we dont panic if a bug creeps in where `targetMemberInfo` is nil but `target` non-empty.

Reply:
I had it in originally but moved it out whilst debugging when I accidentally broke `lxc copy` command since I didn't handle the redirect correctly. Thanks.